cmd/devicesimulator/main.go

```diff
@@ -236,98 +295,169 @@ func createAgentConfigTemplate(dataDir string, configFile string) *agent_config.
 	return agentConfigTemplate
 }
 
-func createAgents(log *logrus.Logger, numDevices int, initialDeviceIndex int, agentConfigTemplate *agent_config.Config) ([]*agent.Agent, []string) {
-	log.Infoln("creating agents")
-	agents := make([]*agent.Agent, numDevices)
-	agentsFolders := make([]string, numDevices)
+func copyAgentFiles(log *logrus.Logger, certDir, agentDir string) {
+	copyDir := func(path string) string {
+		return filepath.Join(agentDir, path)
+	}
+	fileCopyMap := map[string]struct {
+		dir   string
+		files []string
+	}{
+		certDir: {
+			dir:   copyDir(agent_config.DefaultConfigDir),
+			files: []string{"ca.crt", "client-enrollment.crt", "client-enrollment.key"},
+		},
+		// copies the routing tables from /proc/net into the agent's directories so that it will be able to parse
+		// them and select a default route. Currently, it spends 45 seconds attempting to resolve a default route
```

Comment:
the default route and really all the systemInfo is just cosmetic status why not just disable `system-info: []` during simulation. are we using it?

Reply:
You had mentioned that previously and I had tried disabling it that way, but it didn't work. I'll take a look further though